spec/requests/omniauth_callbacks_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Omniauth Callbacks', type: :request do
+  describe 'POST #google_oauth2' do
+    context 'given the email is available' do
+      it 'renders a successful response' do
+        user = Fabricate(:user)
+        Request::OmniAuthCallback.new(:google_oauth2).mock uid: user.uid
```

Comment:
The usage of parenthesis is inconsistent 🙈 Why omit them for the second method call? 

```suggestion
        Request::OmniAuthCallback.new(:google_oauth2).mock(uid: user.uid)
```